src/scale.ts

```diff
@@ -382,9 +382,13 @@ export interface Scale {
    */
   domain?: Domain;
 
+
+  // Hide because we might not really need this.
```

Comment:
So If I understand it correctly, you set the reverse property but don't want to remove it completely because then you would have to store it somewhere else.